fabric_plugin/tests/tests.py

```diff
@@ -526,9 +529,109 @@ def test_run_script_return_value(self):
                     'return_value': expected_return_value
                 },
                 'command_prefix': 'dash'
-            })
+            },
+        )
         self.assertEqual(return_value, expected_return_value)
 
+    def test_crash_abort_after_return(self):
+        try:
+            self._execute(
+                'test.run_script',
+                script_path='scripts/script.sh',
+                process={
+                    'env': {
+                        'test_operation': self._testMethodName,
+                        'return_value': 'some_value'
+                    },
+                    'command_prefix': 'dash'
+                },
+                task_retries=2)
+        except NonRecoverableError, e:
```

Comment:
we tend to prefer the newer `except NonRecoverable as e` in stead of the comma based version